cmd/dex/config.go

```diff
@@ -46,6 +49,9 @@ type Config struct {
 	// querying the storage. Cannot be specified without enabling a passwords
 	// database.
 	StaticPasswords []password `json:"staticPasswords"`
+
+	// Service to do all relevant LDAP operations for a given user.
+	Userinfo Userinfo `json:"userinfo"`
```

Comment:
I would change the struct name, it is not directly the user information, its more a DRDAdapter.

Reply:
I disagree. The way how the whole package userinfo is implemented, with the interface defintion of userinfo, is in a way that the actual implementation is independet. As a matter of fact, I think we may have multiple userinfo backends in the future, for instance, we also might integrate the OIDC userinfo endpoint of Daimler in the future.